### PR TITLE
[tests] Remove a few more usages of NoOpAllBootstrapper from tests

### PR DESCRIPTION
**Pull request by vpranckaitis**
**What this PR does / why we need it**:

Removes a few more usages of `NoOpAllBootstrapper` from integration tests

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note
NONE
```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note
NONE
```

### CODE REVIEW

File: src/dbnode/integration/write_quorum_test.go [hunk 1]
```diff
@@ -223,14 +225,14 @@ func makeTestWrite(
...
-
 	nsOpts := namespace.NewOptions()
 	md, err := namespace.NewMetadata(testNamespaces[0],
 		nsOpts.SetRetentionOptions(nsOpts.RetentionOptions().SetRetentionPeriod(6*time.Hour)))
 	require.NoError(t, err)
 
 	nspaces := []namespace.Metadata{md}
-	nodes, topoInit, closeFn := newNodes(t, numShards, instances, nspaces, false)
+	nodes, topoInit, closeFn := newNodes(t, numShards, instances, nspaces, false,
+		bootstrapper.NoOpAllBootstrapperName)
```
Comment (linasm) [inline src/dbnode/integration/write_quorum_test.go:235]: Any reason why in some cases we can get away with `UninitializedTopology` as final bootstrapper, and in some cases we are still keeping `NoOpAllBootstrapperName`?

File: src/dbnode/integration/write_tagged_quorum_test.go [hunk 2]
```diff
@@ -255,7 +256,8 @@ func makeTestWriteTagged(
 	numShards int,
 	instances []services.ServiceInstance,
 ) (testSetups, closeFn, testWriteFn) {
-	nodes, closeFn, clientopts := makeMultiNodeSetup(t, numShards, true, false, instances)
+	nodes, closeFn, clientopts := makeMultiNodeSetup(t, numShards, true, false,
+		bootstrapper.NoOpAllBootstrapperName, instances)
```
Comment (linasm) [inline src/dbnode/integration/write_tagged_quorum_test.go:260]: Same q as above.

File: src/dbnode/integration/write_quorum_test.go (hunk 1 above)
Comment (vpranckaitis) [reply src/dbnode/integration/write_quorum_test.go:235]: Uninitialized bootstrapper only comes into play when number of initializing shards is higher than leaving ones:
https://github.com/m3db/m3/blob/a9898e234eba88f3b1b03270d6d53c35a1259f1a/src/dbnode/storage/bootstrap/bootstrapper/uninitialized/source.go#L126-L132

However, there are some tests which doesn't satisfy that and fall into retry loop:
https://github.com/m3db/m3/blob/a9898e234eba88f3b1b03270d6d53c35a1259f1a/src/dbnode/integration/fetch_tagged_quorum_test.go#L138-L143

File: src/dbnode/integration/write_tagged_quorum_test.go (hunk 2 above)
Comment (vpranckaitis) [reply src/dbnode/integration/write_tagged_quorum_test.go:260]: See <TODO insert url>

**Review by linasm — APPROVED** [review]

**Pull request merged**